cola/themes.py

```diff
@@ -30,11 +30,12 @@ def build_style_sheet(self, app_palette):
         if self.style_sheet == EStylesheet.CUSTOM:
             return self.style_sheet_custom(app_palette)
         elif self.style_sheet == EStylesheet.FLAT:
+            print(self.style_sheet_flat())
             return self.style_sheet_flat()
         else:
             return self.style_sheet_default(app_palette)
 
-    def build_palette(self, app_palette):
+    def build_palette(self, app_palette) -> QtGui.QPalette:
```

Comment:
Heads-up that I'm going to remove the type annotation here. It's ok in docstrings, but sometimes I do still run this under python2 (I know).

Reply:
I later noticed the other review comments below so I'll let you remove the type annotations when updating this file for those notes.

Reply:
done